vaccine-standby/.env

```diff
@@ -1,4 +1,12 @@
 # description: Your Twilio phone number for sending and receiving messages
 # format: phone_number
 # required: true
-TWILIO_PHONE_NUMBER=
\ No newline at end of file
+TWILIO_PHONE_NUMBER=
+
+# description: Use this password to access your intake form results
+# format: string
+# required: true
+ADMIN_PASSWORD=password
```

Comment:
Need to add this to README

Reply:
Oh shoot, this actually shouldn't be committed because it will be set during the CodeExchange setup.